api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceIntegrationTests.java

```diff
@@ -129,15 +132,29 @@ void shouldAcceptAndCreateANewValidClaimWithMissingAddressFields(String county)
         //Given
         ClaimDTO claim = aClaimDTOWithCounty(county);
         CombinedIdentityAndEligibilityResponse identityAndEligibilityResponse = anIdMatchedEligibilityConfirmedUCResponseWithAllMatches();
-        stubEligibilityServiceWithSuccessfulResponse(identityAndEligibilityResponse);
+        wiremockManager.stubEligibilityResponse(identityAndEligibilityResponse);
 
         //When
         ResponseEntity<ClaimResultDTO> response = restTemplate.exchange(buildClaimRequestEntity(claim), ClaimResultDTO.class);
 
         //Then
         assertThatClaimResultHasNewClaim(response);
         assertClaimPersistedSuccessfully(claim, ELIGIBLE);
-        verifyPostToEligibilityService();
+        wiremockManager.assertThatEligibilityRequestMade();
+    }
+
+    @Test
+    void shouldRejectEligibleClaimantWhenNoChildrensDatesOfBirthMatch() throws JsonProcessingException {
+        ClaimDTO claim = aValidClaimDTOWithExpectedDeliveryDateAndChildrenDob(null, List.of(LocalDate.now().minusDays(1)));
+        CombinedIdentityAndEligibilityResponse identityAndEligibilityResponse = anIdMatchedEligibilityConfirmedUCResponseWithAllMatches();
+        wiremockManager.stubEligibilityResponse(identityAndEligibilityResponse);
+
+        //When
+        ResponseEntity<ClaimResultDTO> response = restTemplate.exchange(buildClaimRequestEntity(claim), ClaimResultDTO.class);
+
+        //Then
+        assertRejectedResponse(response, ELIGIBLE);
```

Comment:
Can we also make sure that the `isPregnantOrAtLeast1ChildMatched` is asserted to be false here too?